### PR TITLE
Retry requests that receive 429

### PR DESCRIPTION
**Pull request by arcticfly**
## Changes
* Retry requests automatically on 429 errors and manually on any error
* Change OutputCell query to upsert
* Move OutputCell to its own folder, define child components in their own files
* Create sharedStrings file, define 429 error string for both client and server

Before:
Screenshot 2023-07-06 at 9 23 12 PM

After:
Screenshot 2023-07-06 at 9 38 17 PM

### CODE REVIEW

**Review by corbt — COMMENTED** [review]
I was imagining was that the retry logic would live in OutputCell. That way we can message the user that a call got rate limited, and we're retrying in X seconds

I think this one also doesn't have a way to let you retry again once it fails 3 times?

**Comment by arcticfly** [comment]
> I was imagining was that the retry logic would live in OutputCell. That way we can message the user that a call got rate limited, and we're retrying in X seconds
> 
> I think this one also doesn't have a way to let you retry again once it fails 3 times?

Good idea.

File: src/components/OutputsTable/OutputCell/ErrorHandler.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      {msToWait > 0 && (
+        <Text color="red.600" fontSize="sm">
+          Retrying in {Math.ceil(msToWait / 1000)} seconds...
+        </Text>
+      )}
+    </VStack>
+  );
+};
+
+function breakLongWords(str: string | null): string {
```
Comment (corbt) [inline src/components/OutputsTable/OutputCell/ErrorHandler.tsx:84]: I assume this is so the text doesn't escape the container? using CSS for this like `word-break: break-all` handle that case more simply? https://css-tricks.com/almanac/properties/w/word-break/#aa-values

**Review by corbt — APPROVED** [review]

File: src/components/OutputsTable/OutputCell/ErrorHandler.tsx (hunk 1 above)
Comment (arcticfly) [reply src/components/OutputsTable/OutputCell/ErrorHandler.tsx:84]: break-all can break normal words as well, which we don't want

**Pull request merged**